backend/src/graphql/resolver/ContributionResolver.test.ts

```diff
@@ -375,6 +375,7 @@ describe('ContributionResolver', () => {
 
       describe('wrong contribution id', () => {
         it('throws an error', async () => {
+          jest.clearAllMocks()
```

Comment:
before each

Reply:
no, if the describe will contain a BeforeEach with jest.clearAllMocks() the logger.mock will be cleared for the test in line 395/400 too. This will have the consequence that the logger will not contain any invocation and no message to check for.